viscstudies/test/src/org/labkey/test/tests/viscstudies/CAVDStudyTest.java

```diff
@@ -80,9 +82,17 @@ public List<String> getAssociatedModules()
         return Arrays.asList("viscstudies");
     }
 
+    @Override
+    protected void doCleanup(boolean afterTest) throws TestTimeoutException
+    {
+        ExperimentalFeaturesHelper.disableExperimentalFeature(createDefaultConnection(), "GWTStudyDesign");
```

Comment:
I know @Trey already approved this, but just curious if this would have worked as well:

```
if (!afterTest)
{
 ExperimentalFeaturesHelper.enableExperimentalFeature(createDefaultConnection(), "GWTStudyDesign");
}
else
{
ExperimentalFeaturesHelper.disableExperimentalFeature(createDefaultConnection(), "GWTStudyDesign");
}
```

Reply:
Possibly. Is that the standard pattern the team follows?

Reply:
👋🏻 Wrong Trey!

Reply:
Sorry "Wrong Trey". 

@labkey-klum not necessarily standard pattern, just a personal preference. If it works as is itis fine to leave.